### PR TITLE
clean up default controller names

### PR DESCRIPTION
**Pull request by abake48**
Updated fault controller name in yaml files and cleaned up gen3.launch.py and kortex_control.launch.py

### CODE REVIEW

**Review by moriarty — APPROVED** [review]

**Review by moriarty — DISMISSED** [review]

**Comment by moriarty** [comment]
@abake48 I think you found a naming an error on my part and opened PR https://github.com/PickNikRobotics/picknik_controllers/pull/9

File: kortex_description/arms/gen3/7dof/config/ros2_controllers.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ controller_manager:
       type: position_controllers/GripperActionController
 
     fault_controller:
-      type: picknik_reset_fault_controller/PicknikResetFaultController
```
Comment (moriarty) [inline kortex_description/arms/gen3/7dof/config/ros2_controllers.yaml]: Are you sure your workspace is up to date and has been cleaned if needed?
It looks like you're changing this back from what it should be to what it was.

**Review by moriarty — CHANGES_REQUESTED** [review]
It should be PicknikResetFaultController
https://github.com/PickNikRobotics/picknik_controllers/pull/9

**Comment by abake48** [comment]
@moriarty This PR is ready for re-review

File: kortex_description/arms/gen3/6dof/config/ros2_controllers.yaml [hunk 2]
```diff
@@ -15,7 +15,7 @@ controller_manager:
       type: position_controllers/GripperActionController
 
     fault_controller:
-      type: picknik_reset_fault_controller/PicknikResetFaultController
+      type: picknik_reset_fault_controller/FaultController
```
Comment (moriarty) [inline kortex_description/arms/gen3/6dof/config/ros2_controllers.yaml]: @abake48 is this PicknikResetFaultController or just FaultController?

Comment (abake48) [reply kortex_description/arms/gen3/6dof/config/ros2_controllers.yaml]: Whoops, forgot the 6dof. It should be `PicknikResetFaultController`

**Review by moriarty — APPROVED** [review]

**Pull request merged**